### PR TITLE
auto OCP-24330

### PR DESCRIPTION
**Pull request by SSshahan**
@yapei  please help to review, thanks!

### CODE REVIEW

File: lib/rules/web/admin_console/4.3/goto.xyaml [hunk 1]
```diff
@@ -132,3 +132,9 @@ goto_machineconfigs_page:
   url: k8s/cluster/machineconfiguration.openshift.io~v1~MachineConfig
 goto_machineconfig_pools_page:
   url: k8s/cluster/machineconfiguration.openshift.io~v1~MachineConfigPool
+
+#crd
+goto_crd_page:
```
Comment (yapei) [inline lib/rules/web/admin_console/4.3/goto.xyaml]: maybe `goto_crds_page`

File: lib/rules/web/admin_console/4.3/goto.xyaml [hunk 2]
```diff
@@ -132,3 +132,9 @@ goto_machineconfigs_page:
...
+  url: k8s/cluster/customresourcedefinitions
+goto_crd_instance:
```
Comment (yapei) [inline lib/rules/web/admin_console/4.3/goto.xyaml]: and `goto_crd_instances_page`

**Comment by SSshahan** [comment]
@yapei could you help to review again?

**Pull request merged**